src/components/CurrentRefinements/__tests__/CurrentRefinements-test.tsx

```diff
@@ -167,9 +169,9 @@ describe('CurrentRefinements', () => {
         ],
       };
 
-      const tree = mount(<CurrentRefinements {...props} />);
+      const { container } = render(<CurrentRefinements {...props} />);
```

Comment:
does container have a method that could be used? seems like that's likely what causes it to be an array

Reply:
The snapshots from `preact-testing-library` are _not_ arrays, while the ones from `enzyme` are.